src/Sulu/Bundle/ContactBundle/Contact/ContactManagerInterface.php

```diff
@@ -21,11 +24,11 @@ interface ContactManagerInterface
     /**
      * adds an address to the entity.
      *
-     * @param string $entity The entity to add the address to
+     * @param ContactAddress $entity The entity to add the address to
```

Comment:
Does not look correctly for me is `AccountInterface` in `AccountManager` and a `ContactInterface` for the `ContactManager` so we should typehint it with `object` or use generics for it.

Reply:
For simple I would go here with `ContactInterface|AccountInterface`